intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -3263,6 +3263,46 @@ def scan_exchange_identifier(field):
     },
 }
 
+# https://www.shadowserver.org/what-we-do/network-reporting/accessible-sip-report/
+scan_sip = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip', validate_ip),
+        ('source.port', 'port', convert_int),
+    ],
+    'optional_fields': [
+        ('protocol.transport', 'protocol'),
+        ('source.reverse_dns', 'hostname'),
+        ('extra.', 'tag', validate_to_none),
+        ('source.asn', 'asn', invalidate_zero),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('source.geolocation.city', 'city'),
+        ('extra.', 'sip', validate_to_none),
+        ('extra.', 'sip_code', validate_to_none),
+        ('extra.', 'sip_reason', validate_to_none),
+        ('user_agent', 'user_agent', validate_to_none),
+        ('extra.', 'sip_via', validate_to_none),
+        ('extra.', 'sip_to', validate_to_none),
+        ('extra.', 'sip_from', validate_to_none),
+        ('extra.', 'content_length', convert_int),
+        ('extra.', 'content_type', validate_to_none),
+        ('extra.', 'server', validate_to_none),
+        ('extra.', 'contact', validate_to_none),
```

Comment:
Would a `sip_` prefix to all the SIP protocol-related fields (server, contact, cseq, call_id, allow) be useful to understand their meaning?

Reply:
The fields have been updated.